spark/v3.1/spark/src/main/scala/org/apache/spark/sql/catalyst/parser/ArcticExtendSparkSqlAstBuilder.scala

```diff
@@ -125,7 +125,6 @@ class ArcticExtendSparkSqlAstBuilder(delegate: ParserInterface)
       case Some(query) =>
         val propertiesMap = buildProperties(primaryForCtas, properties)
         var writeOptions: Map[String, String] = Map.empty
-        writeOptions += ("overwrite-mode" -> "dynamic")
```

Comment:
this option is used in data-frame api(use data-frame api to create keyed table), will it be affected if you delete it ?